src/gl_fns.rs

```diff
@@ -342,7 +342,6 @@ impl Gl for GlFns {
         }
     }
 
-    // FIXME: Does not verify buffer size -- unsafe!
```

Comment:
Also, this is still true (we don't verify that there is enough data on the buffer for the given `format`, `width` and `height`), so please add these comments back.